students/BRACH_Jakub/lesson_01_basics/L01P17_Digital_Clock.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+import math
+
+print('Enter minutes')
+minutes = int(input())
+
+#remove full days
+minutes = minutes % (24*60)
+
+hours = minutes / 60
+minutes = minutes % 60
+
+print('The time is {0:d}:{1:02d}'.format(hours, minutes))
```

Comment:
This line throws an error: `ValueError at line 8: Unknown format code 'd' for object of type 'float'`
After division var `hours` is type of `float`.

This code print wrong output. 
e.g. for value 150, output should be "2 30", where with this code it's "2.5 30"

Reply:
Done